services/connected_account.yaml

```diff
@@ -91,14 +73,14 @@ Resources:
         - Key: cloudzero-stack
           Value: !Ref AWS::StackName
         - Key: cloudzero-reactor-account-id
-          Value: !Ref ReactorAccountId
+          Value: !FindInMap [ CallbackConfiguration, prod, ReactorAccountId ]
       Parameters:
         IsResourceOwnerAccount: !GetAtt Discovery.Outputs.IsResourceOwnerAccount
         IsAuditAccount: !GetAtt Discovery.Outputs.IsAuditAccount
         IsMasterPayerAccount: !GetAtt Discovery.Outputs.IsMasterPayerAccount
         IsCloudTrailOwnerAccount: !GetAtt Discovery.Outputs.IsCloudTrailOwnerAccount
         ExternalId: !Ref ExternalId
-        ReactorAccountId: !Ref ReactorAccountId
+        ReactorAccountId: !FindInMap [ CallbackConfiguration, prod, ReactorAccountId ]
```

Comment:
fine for initial testing, not worth blocking the PR but we should discuss how to improve this for later

Reply:
agreed